### ISSUE TITLE
Clean-up page--- templates

### ISSUE THREAD

**Issue by PBruk** [MITLibraries/MITLibraries-child#79]
Some of the code was written in haste (by me) and is inefficient and needlessly repetitive.

**Comment by matt-bernhardt** [comment]
I agree with making code non-repetitive, and in cleaning up code - but I'm finding this a bit too vague to be really actionable. Can you edit the description of the issue more specific? Are there templates that you think need attention the most, or is there a certain kind of problem that needs to be corrected?

**Comment by PBruk** [comment]
Matt, good question. It was actually your review of one of my PRs which forced me to rethink my template re-write, which I am referring to here regarding these other templates. Here is an example of what I am referring to:

![page php](https://cloud.githubusercontent.com/assets/1832277/16788108/f15df44e-486f-11e6-8a22-34ae86b85a88.png)

Instead of this:

![page-front php](https://cloud.githubusercontent.com/assets/1832277/16788116/01361ae0-4870-11e6-8cff-fc9c505a9059.png)
